libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -125,61 +125,72 @@ const AP_Param::GroupInfo AP_MotorsHeli_RSC::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("THRCRV_100", 12, AP_MotorsHeli_RSC, _thrcrv[4], AP_MOTORS_HELI_RSC_THRCRV_100_DEFAULT),
 
-    // @Param: GOV_SETPNT
-    // @DisplayName: Rotor Governor Setpoint
-    // @Description: Main rotor rpm setting that governor maintains when engaged.  Set to the rotor rpm your helicopter runs in flight. When a speed sensor is installed the rotor governor maintains this speed. For governor operation this should be set 10 rpm higher than the actual desired headspeed to allow for governor droop
+    // @Param: GOV_RPM
+    // @DisplayName: Rotor RPM Setting
+    // @Description: Main rotor RPM that governor maintains when engaged
     // @Range: 800 3500
     // @Units: RPM
     // @Increment: 10
     // @User: Standard
-    AP_GROUPINFO("GOV_SETPNT", 13, AP_MotorsHeli_RSC, _governor_reference, AP_MOTORS_HELI_RSC_GOVERNOR_SETPNT_DEFAULT),
+    AP_GROUPINFO("GOV_RPM", 13, AP_MotorsHeli_RSC, _governor_rpm, AP_MOTORS_HELI_RSC_GOVERNOR_RPM_DEFAULT),
```

Comment:
Need to make this a new parameter rather than re-naming the existing parameter especially since it changed type (AP_INT16 to AP_Float)